### PR TITLE
added styles for section structure

### LINKED ISSUE #31: About Us - Structure Page styling
Add styles for section + default wrapper in structure page

### PR DESCRIPTION
**Pull request by rrrragas**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #31 

Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.live/_drafts/sagar/structure
- After: https://31-about-us-structure-page-styling--sunstar--hlxsites.hlx.live/_drafts/sagar/structure

- [ ]  If you are adding a new block or a variation to an existing block, has it been added in the [block-library](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/sunstar/sunstar/sidekick/library.xlsx?d=w7f8a415fe192444caa87806b565bb62d&csf=1&web=1&e=oJWCS0) ?

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -406,6 +406,14 @@ main img {
 
 /* START Section Styles START */
 
+main .section .section-container h6 {
+  margin-bottom: 0.5rem;
+}
+
+main .section .section-container p {
```
Comment (sdmcraft) [inline styles/styles.css:413]: Is this universally applicable in the original site i.e. majority paragraphs needing this much padding?

File: styles/styles.css [hunk 2]
```diff
@@ -678,6 +686,18 @@ main .section.thank-you p {
   font-weight: var(--font-weight-medium);
 }
 
+main .section.structure {
```
Comment (sdmcraft) [inline styles/styles.css:689]: How about reusing one of the existing section variants here like `narrow` ? I'd prefer to reuse if there are minor differences in styling.

File: styles/styles.css (hunk 1 above)
Comment (rrrragas) [reply styles/styles.css:413]: yes, checked across few pages and observed this common pattern hence added it as global rule.

File: styles/styles.css (hunk 2 above)
Comment (rrrragas) [reply styles/styles.css:689]: existing sections not matching with width and text alignment properties. at the moment not sure if we need to update narrow section definition (may be existing narrow width works well in other pages) or to define a new section. Please let me know if we need to park this rule for time being.

Comment (sdmcraft) [reply styles/styles.css:689]: Could you share URL with narrow section style?

Comment (rrrragas) [reply styles/styles.css:689]: https://main--sunstar--hlxsites.hlx.page/_drafts/sagar/structure-with-narrow-section

Comment (sdmcraft) [reply styles/styles.css:689]: I guess we can add a few `spacer` blocks in the word doc to achieve the needed gaps.

File: styles/styles.css (hunk 1 above)
Comment (sdmcraft) [reply styles/styles.css:413]: Could you check with [global styles of the live site](https://www.sunstar.com/wp-content/themes/sunstar-group/public/css/app.css)? I see the following global style in it 
```
p {
    margin-top: 0;
    margin-bottom: 1rem
}
```

File: styles/styles.css (hunk 2 above)
Comment (dnbute) [reply styles/styles.css:689]: There also seems to be a `spaced` section style

**Comment by rrrragas** [comment]
it is a old PR. will be creating a new pr with latest changes

**Pull request closed without merging**